eras/byron/ledger/impl/test/Test/Cardano/Chain/Block/Model.hs

```diff
@@ -273,7 +273,7 @@ classifyTransactions =
 invalidChainTracesAreRejected ::
   TestLimit ->
   [(Int, SignalGenerator CHAIN)] ->
-  ([PredicateFailure CHAIN] -> ChainValidationError -> PropertyT IO ()) ->
+  (NonEmpty (PredicateFailure CHAIN) -> ChainValidationError -> PropertyT IO ()) ->
```

Comment:
It is worth pointing out that a change like this requires two things:
* changelog entry
* major version bump, since it is  breaking change
* lower bounds update on the `small-test` package version (See also my comment on that package)
* lower bound update on any package affected by this change (which I don't think there are any in this repo)

More info can be found on the process in the [RELEASING document](https://github.com/IntersectMBO/cardano-ledger/blob/master/RELEASING.md)

Reply:
The changes in `eras/byron/ledger/impl/test` are in functions that aren't exported from their modules, so I don't think they need a changelog entry.

However, I'll add/update lower bounds for `small-steps` and `small-steps-test` anywhere they're used.

Reply:
Byron has a slightly different structure and this module is actually a publicly visible module: https://github.com/IntersectMBO/cardano-ledger/blob/4223cd483af91913c356a882a1ea1ff4357fe3fc/eras/byron/ledger/impl/test/cardano-ledger-byron-test.cabal#L118

Reply:
The module is visible, but AFAICT `invalidChainTracesAreRejected` and `invalidSizesAreRejected` aren't exported from the module

Reply:
Oh yeah, sorry. You are totally right. I misread your first message